modules/admin_manual/pages/configuration/files/encryption/enabling-user-key-encryption.adoc

```diff
@@ -57,7 +58,7 @@ sudo -u www-data php occ maintenance:singleuser --off
 ....
 
 [[how-to-enable-users-file-recovery-keys]]
-== How To Enable User File Recovery Keys
+== How To Enable Users File Recovery Keys
```

Comment:
This change doesn't read correctly to me.

Reply:
Will check if the anchor is used somewhere and change it accordingly

Reply:
The anchor's not the issue, it's the change of "User" to "Users". I don't believe that this is grammatically correct in this context (but will double-check).

Reply:
After checking it, it's grammatically correct, however I don't see the need for the change.